### PR TITLE
fix(ibm): fix broken log statement when attaching floating ip

### PR DESCRIPTION
**Pull request by a-dubs**
floating ip attachment no longer returns the floating ip but rather a boolean representing whether it succeeded or not. This commit updates the code to handle this change properly.

---

Testing of changes:
Ran updated `examples/ibm.py` script and ensured floating IP attached successfully without raising an error and that the expected floating IP name appeared in the log.

### CODE REVIEW

**Comment by a-dubs** [comment]
didn't update version file until this got a +1 and is ready to merge.

File: pycloudlib/ibm/instance.py [hunk 1]
```diff
@@ -681,7 +681,7 @@ def _attach_floating_ip_until_success(
                 )
         self._log.info(
             "Successfully attached floating ip: %s",
-            attached_floating_ip["name"],
```
Comment (blackboxsw) [inline pycloudlib/ibm/instance.py:684]: Got it; attached floating_ip is a bool. +1. and use_existing_floating_ip_w_name renamed.

**Review by blackboxsw — APPROVED** [review]
LGTM! VERSION bump and ship it.

**Comment by a-dubs** [comment]
@blackboxsw tysm for the review!!! version hath been bumped :saluting_face:

**Pull request merged**